source/SIL.AppBuilder.Portal.Frontend/src/lib/fetch.ts

```diff
@@ -2,8 +2,8 @@ import { getToken } from './auth0';
 import { getCurrentOrganizationId } from './current-organization';
 
 export const tryParseJson = async (response) => {
-  const text = await response.text();
 
+  const text = await response.text();
```

Comment:
what's with this change?

Reply:
I can't tell what's different? lol

Reply:
oh, it changed lines? weird

Reply:
I didn't change this one... just added some debugger and space